taprpc/taprootassets.proto

```diff
@@ -460,6 +460,10 @@ message ListBalancesResponse {
 }
 
 message ListTransfersRequest {
+    // anchor_tx_hash specifies the hash of the anchor transaction for which to
+    // retrieve transfers. An empty value indicates that this parameter should
+    // be disregarded in transfer selection.
+    bytes anchor_tx_hash = 1;
```

Comment:
I'm not sure if we should define this as a string. Otherwise when using REST, you would need to reverse the byte order in the URL. Or perhaps we can do:
```
oneof anchor_tx {
   bytes anchor_tx_hash = 1;
   string anchor_txid_str = 2;
}
```

And then in the REST mapping we could do:
```
    - selector: taprpc.TaprootAssets.ListTransfers
      get: "/v1/taproot-assets/assets/transfers"
      additional_bindings:
        - get: "/v1/taproot-assets/assets/transfers/{anchor_tx.anchor_txid_str}"
```